sygnal/apnspushkin.py

```diff
@@ -151,7 +160,7 @@ def __init__(self, name, sygnal, config):
                 loop=loop,
             )
 
-            self._report_certificate_expiration(certfile)
+            self._report_certificate_expiration(cast(str, certfile))
```

Comment:
We _could_ try something fancy with `get_config` so that it does the validation and returns the correct type at the same time?

```py
def get_config(key: str, type_: Type[T], default: DefaultT = None) -> Union[T, DefaultT]

...

certfile = self.get_config("certfile", str)
```